ipc_sendfile.c

```diff
@@ -0,0 +1,303 @@
+////////////////////////////////////////////////////////////////////////////////
+//IPC SENDER
+//Message passing and shared memory are written according to the example provided by QNX training
+//
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <sys/neutrino.h>
+#include <sys/netmgr.h>     // #define for ND_LOCAL_NODE is in here
+#include "ipc_common.h"
+#include <sys/iofunc.h>
+#include <sys/dispatch.h>
+
+#include <getopt.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+
+#include <string.h>
+
+int m_buff_size = 4096;
+int p_buff_size = 4096;
+void ipcMessagePassingSend(int fd);
+void ipcSharedMemorySend(int fd, int buffer_size);
+
```

Comment:
You need to declare each function which is after main()

Reply:
All relevant things have been modified. Could you check it now?